V2 callout (#311)
ueberdosis/tiptap-docs: src/content/ui-components/components/blockquote-button.mdx

---

import { CodeDemo } from '@/components/CodeDemo'
import { Callout } from '@/components/ui/Callout'

<Callout title="Upgrade to v3" variant="warning">

>   To use UI components or templates you need to upgrade to Editor version 3.x. See the [upgrade

Reviewer: ```suggestion
  Upgrade to Tiptap 3 to start using UI components and templates. See the [upgrade
```
@ThisDavidRichard  I re-wrote the copy, do you like it better?
